x-pack/plugins/security_solution/public/entity_analytics/common/utils.ts

```diff
@@ -26,6 +27,27 @@ export const RISK_SEVERITY_COLOUR: { [k in RiskSeverity]: string } = {
   [RiskSeverity.Critical]: SEVERITY_COLOR.critical,
 };
 
+export const useRiskSeverityColors = (): { [k in RiskSeverity]: string } => {
+  const { euiTheme } = useEuiTheme();
+
+  return {
+    [RiskSeverity.Unknown]: euiTheme.colors.vis.euiColorVisNeutral0, // TODO: this is a closest guess based on severity colors, change to grey20 when available
+    // TODO: update these with V9.0.0 severity palette colors when available / keep if the below are  updated with the palette
+    [RiskSeverity.Low]: euiTheme.flags.hasVisColorAdjustment
+      ? '#54B399'
+      : euiTheme.colors.vis.euiColorVisSuccess0,
+    [RiskSeverity.Moderate]: euiTheme.flags.hasVisColorAdjustment
+      ? '#D6BF57'
+      : euiTheme.colors.vis.euiColorVis9,
+    [RiskSeverity.High]: euiTheme.flags.hasVisColorAdjustment
+      ? '#DA8B45'
+      : euiTheme.colors.vis.euiColorVisWarm1,
+    [RiskSeverity.Critical]: euiTheme.flags.hasVisColorAdjustment
+      ? '#F66D64'
```

Comment:
This seems to be a mistake, it was `#E7664C` before. But as mentioned above, we can use tokens instead.